src/Control/Optics/Linear.hs

```diff
@@ -0,0 +1,15 @@
+module Control.Optics.Linear
+  ( Optic_(..)
+  , Optic
+  , module Control.Optics.Linear.Iso
+  , module Control.Optics.Linear.Lens
+  , module Control.Optics.Linear.Prism
+  , module Control.Optics.Linear.Traversal
```

Comment:
Please put in haddock headings here and make sure it looks nice with the inner headings having two stars `-- **`

Reply:
Actually, nevermind, I'll do that in my documentation PR anyway.

Reply:
Yes, there was no documentation here.

In this particular setup, though, the idea is that the main module simply contains links to the various other modules (the Haddock will just say, `module Iso` etc…)

Each of these module, on the other hand, is supposed to have a well-sectioned organisation.

I used this organisation, which is the same as that of the lens library, because optics module tend to get large, hence separating the exports into file may make sense.

But if we feel that we prefer having a single module with sections instead, I'm fine with it too. I don't have a strong opinion either way.